Resources/Public/JavaScript/theodia-more.js

```diff
@@ -32,13 +32,17 @@ document.addEventListener('DOMContentLoaded', function () {
                 body: formData
             }).then(response => response.json())
                 .then(data => {
+                    if (data.html) {
+                        targetEl.innerHTML += data.html;
+                        targetEl.dataset.events = data.numberEvents;
+                    }
                     if (!data.hasMore) {
```

Comment:
To me this check is wrong and the lines

```
targetEl.innerHTML = data.html
targetEl.dataset.events = data.numberEvents;
```

are supposed to go outside of the `if`

but I don't get your patch.